components/Radio/RadioWrapper.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { SFC } from "react"
+import { RadioWrapperProps } from "./Interface"
+
+const RadioWrapper: SFC<RadioWrapperProps> = (props = defaultProps) => {
+  const {
+    prefixCls,
+    className,
+    style,
+    disabled,
+    readOnly,
+    tabIndex,
+    onClick,
+    onFocus,
+    onBlur,
+    autoFocus,
+    onChange,
+    value,
+    groupValue,
+    setValue,
+    ...others
+  } = props
+
+  const isChecked = () => value === groupValue || props.defaultChecked
+
+  const handleChange = e => {
+    if (props.disabled) {
+      return
+    }
+    if (setValue) {
+      setValue(value)
+    }
+    if (onChange) {
+      onChange({
+        target: {
+          ...props,
+          checked: e.target.checked,
+        },
+        stopPropagation() {
+          e.stopPropagation()
+        },
+        preventDefault() {
+          e.preventDefault()
+        },
+        nativeEvent: e.nativeEvent,
+      })
+    }
+  }
+
+  const globalProps = Object.keys(others).reduce((prev, key) => {
```

Comment:
Add comments explaining this, and put a link to the snippet where you copied from.

Reply:
Done